docs/source/nlp/usage_guides/nlp_properties_extended.rst

```diff
@@ -163,3 +164,52 @@ Michael Jordan (1963) is an American former professional basketball player and b
 Michael Jordan (1963) is an American former professional basketball player and businessman. When was Michael born?      Michael Jeffrey Jordan was born in 1963                0.5
 Michael Jordan (1963) is an American former professional basketball player and businessman. What did he achieve?        He won many NBA championships with the Chicago Bulls   0.0
 ======================================================================================================================  =====================================================  ===================
+
+
+Relevance
+---------
+
+The Relevance score is a measure of how relevant the LLM output is to the input, ranging from 0 (not relevant) to 1
+(fully relevant).
```

Comment:
Can I not read the entire thing and just tell me what you changed? 😅

Reply:
lol not a lot